Source/Fuse.Common/Internal/MiniList.uno

```diff
@@ -3,6 +3,15 @@ using Uno.Collections;
 
 namespace Fuse.Internal
 {
+	//An error with Uno handling the Enumerator<T> generic is preventing this from being a private enum to MiniList
+	//E2047: No implicit cast from Fuse.Internal.MiniList<T>.Mode to Fuse.Internal.MiniList<T>.Mode
```

Comment:
No. I didn't find the problem significant enough (doesn't touch a public API, minor annoyance) to be side-tracked coming up with a smaller reproduction at this time.